### ISSUE TITLE
types errors in core.d.ts

### ISSUE THREAD

**Issue by miao17game** [ws-node/Bonbons#2]
```NODE
node_modules/bonbons/dist/metadata/core.d.ts(1,8): error TS1192: Module '"C:/Users/sWX458266/Desktop/tsc/node_modules/@types/express/index"' has no default export.
```

```TYPESCRIPT
import express from "express";  // <---- PROBLEM HERE
import * as core from "express-serve-static-core";
```

**Comment by miao17game** [comment]
problem is in core.ts file:

```TYPESCRIPT
import * as express from "express";
import * as core from "express-serve-static-core";
```

**Comment by miao17game** [comment]
need tsc recompile

**Comment by miao17game** [comment]
fix in 86b2f3920c3e585fa23f80cd4ebcc0ad7277be6c
